unit/Makefile

```diff
@@ -3,7 +3,6 @@
 # Source files for test utilities
 SRC = src/expr/require_expr.cpp \
       src/ansi-c/c_to_expr.cpp \
```

Comment:
These files should probably also be moved into the `testing-utils`

Reply:
Moved. It looks like none of the tests actually use these files though, is this correct?